Gemfile

```diff
@@ -5,7 +5,7 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in pipefy_message.gemspec
 gemspec
 
-gem "activesupport", "~> 5.0", ">= 5.0.0.1"
+gem "activesupport", "~> 4.2.11.3"
```

Comment:
Just wondering, why do we need ActiveSupport, and why are we using an old version of it?

Reply:
The version I forgot to back to the last version, We used to create objects from strings in real time with `.constantize.new` [here](https://github.com/pipefy/pipefy_message/blob/a30e74009b9385f04734b4342224a978c45281bb/lib/pipefy_message/consumer.rb#L59)

I changed the version to test one thing in core and forgot to come back the version

Reply:
Got it, thanks @eHattori 

But now that we are on Rails 4, does it make sense to use the Rails 5 version of it?

Reply:
done!